spec/requests/v1/pacbio/runs_spec.rb

```diff
@@ -23,6 +34,16 @@
       patch v1_pacbio_run_path(run), params: body, headers: json_api_headers
       expect(response).to have_http_status(:success), response.body
     end
+
+    it 'publishes volume tracking message for each used aliquot' do
+      allow(Emq::Publisher).to receive(:publish)
+
+      patch v1_pacbio_run_path(run), params: body, headers: json_api_headers
+      expect(response).to have_http_status(:success), response.parsed_body
+
+      run = Pacbio::Run.first
+      expect(Emq::Publisher).to have_received(:publish).with(array_including(run.all_library_aliquots), instance_of(Pipelines::Configuration::Item), 'volume_tracking') # rubocop:disable RSpec/MessageSpies
```

Comment:
As above. 

I think we only need to test that a publish message has been received. I presume the publish message is also tested elsewhere.